pkg/session_recording/recordings.jsx

```diff
@@ -203,7 +205,14 @@
         }
 
         goBackToList() {
+            console.log(cockpit.location.options);
+            if(cockpit.location.options.id) {
+                delete cockpit.location.options.id;
+                cockpit.location.go([], cockpit.location.options);
+                return;
+            }
             cockpit.location.go('/');
+            return;
```

Comment:
Please make it an `if` with two branches and without "return".
I.e.:

    if (cockpit.location.options.id) {
        ...
    } else {
        ...
    }

This way it will be easier to read and understand.